pennylane/ops/op_math/prod.py

```diff
@@ -226,6 +227,16 @@ def is_hermitian(self):
                 return False
         return all(op.is_hermitian for op in self.factors)
 
+    @property
+    def overlapping_wires(self) -> bool:
+        """Boolean expression that indicates if the factors have overlapping wires."""
+        if self._overlapping_wires is None:
+            wires = []
+            for op in self.factors:
+                wires.extend(list(op.wires))
+            self._overlapping_wires = len(wires) != len(set(wires))
```

Comment:
```suggestion
            self._overlapping_wires = (len(wires) != len(set(wires)) )
```

Reply:
This parentheses are not needed (they are removed by `black`)